test/conformance/conformance_mutex.cpp

```diff
@@ -39,21 +39,22 @@ TEST_CASE("Basic Locable requirement test") {
     GeneralTest<oneapi::tbb::spin_mutex>("Spin Mutex");
     GeneralTest<oneapi::tbb::spin_rw_mutex>("Spin RW Mutex");
     GeneralTest<oneapi::tbb::queuing_mutex>("Queuing Mutex");
-    GeneralTest<oneapi::tbb::queuing_rw_mutex>("Queuing RW Mutex");
     GeneralTest<oneapi::tbb::mutex>("Adaptive Mutex");
     GeneralTest<oneapi::tbb::rw_mutex>("Adaptive RW Mutex");
     // TODO: Consider adding Thread Sanitizer (note that accesses inside the transaction
     // considered as races by Thread Sanitizer)
 #if !__TBB_USE_THREAD_SANITIZER
     GeneralTest<oneapi::tbb::speculative_spin_mutex>("Speculative Spin Mutex");
     GeneralTest<oneapi::tbb::speculative_spin_rw_mutex>("Speculative Spin RW Mutex");
+    GeneralTest<oneapi::tbb::queuing_rw_mutex>("Queuing RW Mutex");
+    TestNullMutex<oneapi::tbb::null_mutex>("Null Mutex");
+    TestNullMutex<oneapi::tbb::null_rw_mutex>("Null RW Mutex");
```

Comment:
Could you please clarify why these test cases were moved under !tsan macro?

Reply:
Reverting these changes. I was using ThreadSanitizer and it was throwing a few data race warnings. But these warnings pop up with previous version of doctest too. So I have reverted to the original status.